github_flow.md

```diff
@@ -25,11 +25,13 @@ In short words, we are forking repository from Code for Poznan organisation to o
 
 1. create your own branch 
   * `git checkout -b name_of_your_branch`
-2. make changes in the code
-3. add the changes
+2. push the current branch on github
+  * `git push —set-upstream origin name_of_your_branch`
```

Comment:
As @w1stler pointed out on gitter this one is not necessary and will just create new branch on github that point to the same place as `master`, but if it will help newbies to understand what's going on, I'm not against it.